FAQ.md

```diff
@@ -4,52 +4,207 @@ status: draft
 
 # Frequently Asked Questions (FAQs)
 
-## Why create a new foundation?
-
-There are already several foundations which host open source projects,
-and each has its own focus and requirements.
-The CF is a home for self-governing, code-centric open source projects.
-Each open source project has its own unique needs and goals,
-and not every foundation is the right fit for every project.
-
-There are many critical application frameworks and libraries within the open source ecosystem that are not currently part of a foundation.
-The CF is a new option for projects to consider when looking for a foundation to call home,
+## What is the Commonhaus Foundation?
+
+The Commonhaus Foundation is a non-profit organization dedicated to supporting the long-term
+sustainability of open source projects.
+By offering a neutral home for succession planning, a minimal governance model,
+and fiscal sponsorship,
+it aims to ensure these projects remain accessible and continue to thrive
+for future generations of developers and businesses.
+
+## What led to the creation of the Commonhaus Foundation?
+
+The Commonhaus Foundation was inspired by recognizing specific unmet needs within the open source community.
+Especially for self-governing and code-centric open source projects.
```

Comment:
This isn't a sentence.

Reply:
I've tweaked it. Is it better now?

And thanks for the prompt review!

Reply:
> Is it better now?

Yes.